cluster.tf

```diff
@@ -0,0 +1,42 @@
+# GKE cluster
+resource "google_container_cluster" "primary" {
+  name     = local.resource_name
+  location = data.google_compute_zones.available.region
+
+  remove_default_node_pool = true
+  initial_node_count       = 1
+  network                  = data.ns_connection.network.outputs.vpc_name
+  subnetwork               = data.ns_connection.network.outputs.public_subnet_names[0]
```

Comment:
Is this where the control plane is launched or where the worker nodes are launched?

Reply:
If worker nodes, we should place this in the private subnet
Also, we should default the network to create only 1 private and 1 public subnet

Reply:
Yes, it's a worker node and moved that to the private subnet.

Network will create one public and one private subnet by default https://github.com/nullstone-modules/gcp-network/pull/3

Reply:
👍